mmdet/models/roi_heads/bbox_heads/bbox_head.py

```diff
@@ -640,9 +640,11 @@ def refine_bboxes(self, sampling_results: Union[List[SamplingResult],
             cls_scores = self.loss_cls.get_activation(cls_scores)
         if cls_scores.numel() == 0:
             return None
-
-        labels = torch.where(labels == self.num_classes,
-                             cls_scores[:, :-1].argmax(1), labels)
+        if cls_scores.shape[-1] > self.num_classes:
+            # remove background class
```

Comment:
should we also assert the shape convention like `assert cls_scores.shape[-1] == self.num_classes + 1`?